example/https.js

```diff
@@ -32,14 +32,10 @@ repos.on('fetch', (fetch) => {
     console.log('fetch ' + fetch.commit);
     fetch.accept();
 });
-const port = process.env.PORT || 7005;
 
-var options = {
-  port: 7006, //HTTPS Port
-	key: fs.readFileSync('./privatekey.pem'), //Private Key For HTTPS
-	cert: fs.readFileSync('./certificate.pem') //Certificate For HTTPS
-};
-
-repos.listen(port, options, (error, result, port, protocol) => {
+repos.listen({ httpPort: 7005, httpsPort:7006, key: fs.readFileSync('./privatekey.pem'), cert: fs.readFileSync('./certificate.pem')}, (error, result) => {
```

Comment:
I was suggesting that we just make this

```js
repos.listen({ 
   port: 7005,
   type: 'https',
   key: fs.readFileSync('./privatekey.pem').
   cert: fs.readFileSync('./certificate.pem')
}
```

Does this not fulfill your requirements of knowing exactly what server you would want to start up?

Reply:
No That Look Much Better Than What I Had. I Was Just Not Understanding What You Meant. I Will Add That Now.

Reply:
@oscartbeaumont it could even look like

```
repos.listen('https', { 
   port: 7005,
   key: fs.readFileSync('./privatekey.pem').
   cert: fs.readFileSync('./certificate.pem')
}); 
```

The first example would make it much easier to just deconstruct the options and have one argument being passed in.

Reply:
Ok I Have Fixed The Test, Updated The Docs And Changed The Listen Function To What You Said Above. Note: For The Docs I Did Not Update 'docs/index.html' because It would error if you can change it change the 'repos.listen' so that it works and also add "const path = require('path');" To near the top of the file.

Reply:
Everything Should Be In The Format You Said Above

Reply:
You Should Also try And Fix Your Build System because it says 'All checks have failed' even though it reports that the test were all successful.